tests/phpunit/tests/html-api/wpHtmlProcessorBreadcrumbs.php

```diff
@@ -37,14 +37,26 @@ public function test_navigates_into_normative_html_for_supported_elements( $html
 	public function data_single_tag_of_supported_elements() {
 		$supported_elements = array(
```

Comment:
The naming of the function and the variable is slightly misleading now, since IIUC, the newly added elements are still _un_supported, no? 🤔 (They do throw `WP_HTML_Unsupported_Exception`.)

Reply:
This should be the supported tags. If you compare it with `data_unsupported_special_in_body_tags` from the wpHtmlProcessor suite, there should be no overlap between elements in this array and that array.

Are you sure these tags throw?

A lot of tags move from unsupported to supported in this change because the catch-all "any other tag" rules are implemented here, and only the tags that require (currently unimplemented) special handling remain unsupported.